docs/source/executing-operations/demand-control.mdx

```diff
@@ -0,0 +1,554 @@
+- Attach cost information to spans.
+- Generate log messages whenever the cost delta between estimated and actual is greater than an arbitrary value.
+
+### Instruments
+
+| Instrument      | Value | Default | Description                                                |
+| --------------- | ----- | ------- | ---------------------------------------------------------- |
+| cost.actual     |       |         | The actual cost of an operation, measured after execution.  |
+| cost.estimated  |       |         | The estimated cost of an operation before execution.        |
+| cost.delta      |       |         | The difference between the actual and estimated cost.       |
+
+### Attributes
+
+Attributes for `cost` can be applied to instruments, spans, and events—anywhere `supergraph` attributes are used.
+
+| Attribute      | Value | Default | Description                                                |
+| --------------- | ----- | ------- | ---------------------------------------------------------- |
+| cost.actual     | boolean      |         | The actual cost of an operation, measured after execution.  | 
+| cost.estimated  | boolean       |         | The estimated cost of an operation before execution.        |
+| cost.delta      | boolean      |         | The difference between the actual and estimated cost.       |
+| cost.result      | boolean      |         | The return code of the cost calculation. `COST_OK` or an [error code](../errors/#demand-control)      |
+
+### Selectors
+
+Selectors for `cost` can be applied to instruments, spans, and events—anywhere `supergraph` attributes are used.
+
+| Key  | Value      | Default | Description                                        |
+| ---- | ---------- | ------- | -------------------------------------------------- |
+| cost | estimated, actual, delta, result |         | The estimated, actual, or delta cost values, or the result string |
+
+
+#### Examples
+
+- Enable a `cost.estimated` instrument with the `cost.result` attribute:
+
+```yaml
+telemetry
+  instrumentation:
+    instruments:
+      supergraph:
+        cost.estimated: true
+	  attributes:
+            cost.result: true
+            graphql.operation.name: string
+```
+
+- Enable the `cost.estimated` attribute on `supergraph` spans:
+
+```yaml
+telemetry
+  instrumentation:
+    spans: 
+      supergraph: 
+        attributes:
+          cost.estimated: true
+```
+
+- Log an error when `cost.delta` is greater than 1000:
+
+```yaml
+telemetry
+  instrumentation:
+    events: 
+      supergraph:
+        COST_DELTA_TOO_HIGH:
+          message: "cost delta high"
+          on: response
+          level: error
+          condition: 
+            gt:
+              - cost: delta
+              - 1000
+          attributes:
+            graphql.operation.name: true
+            cost.delta: true
+```
+
+## Demand control workflow
```

Comment:
This guide feels like it should be above the calculation, configuration, and telemetry sections which are more reference materials.